static/app/views/explore/content.tsx

```diff
@@ -34,6 +37,15 @@ export function ExploreContent({}: ExploreContentProps) {
   const navigate = useNavigate();
   const organization = useOrganization();
   const {selection} = usePageFilters();
+  const [groupBys] = useGroupBys();
+  const [visualizes] = useVisualizes();
+  const [resultsMode] = useResultMode();
+
+  const fields = useMemo(() => {
+    return [...groupBys, ...visualizes.flatMap(visualize => visualize.yAxes)].filter(
+      Boolean
+    );
+  }, [groupBys, visualizes]);
```

Comment:
Is there a reason `groupBys` needs to be included here? The `groupBy` is always span attribute, not an aggregation, thus should always be a subset of the fields already available in the autocompletion.